config/initializers/logstash.rb

```diff
@@ -0,0 +1,16 @@
+LogStashLogger.configure do |config|
+  config.customize_event do |event|
+    event["application"] = Settings.application_name
```

Comment:
Should this be `application_name`

Reply:
Yes! Thanks, it started out as `application_name`, but then wanted something shorter.